src/v/kafka/server/connection_context.cc

```diff
@@ -302,6 +302,13 @@ connection_context::dispatch_method_once(request_header hdr, size_t size) {
                             });
                       })
                       .handle_exception([self](std::exception_ptr e) {
+                          // ssx::spawn_with_gate already caught shutdown-like
```

Comment:
Would be good to get a second pair of eyes on this: I don't think the shutdown_input call is necessary to progress shutdown of the overall service, right?

Reply:
I'm not sure. But I think it best that the connection is shutdown cleanly in any case.